hanko-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@teamhanko/hanko-elements",
-  "version": "0.0.3-alpha",
+  "version": "0.0.6-alpha",
```

Comment:
Can you update the lockfile accordingly? I guess its main concern is third party dependency versions but it also keeps track of the root package version and it just doesn't feel right when they're out of tune.